### DISCUSSION TITLE
I keep getting rejects

### DISCUSSION THREAD

**Discussion by munlicode** [community/community#145121] [GitHub Education]
## Hi everyone,
**### Problem**
![problem_all](https://github.com/user-attachments/assets/41713b34-ed78-4c8c-b315-db0692c958c7)

###  **Document from school that I am a student**
![proof](https://github.com/user-attachments/assets/f04e6863-8fa3-460d-b4d4-7495945d257f)

### **GitHub** 

**Public Name**
![public_name](https://github.com/user-attachments/assets/3e9ae77c-d06c-4b67-9139-73009c402a72)

**Billing Name**
![billing_name](https://github.com/user-attachments/assets/549e88b1-331a-44e5-a2cc-bc69568d02b6)

**If it is okay can I replace my name with English translation?**

**Comment by pierreparre** [comment]
hi,
same here i keep getting rejected.
